File: MaterialProviders_stop/AwsCryptographyPrimitives/runtimes/python/test/internal_generated_dafny/TestAwsCryptographyPrimitivesAesKdfCtr.py
import sys
from typing import Callable, Any, TypeVar, NamedTuple
from math import floor
from itertools import count

import module_
import _dafny
import System_
import Wrappers
import BoundedInts
import StandardLibrary_mUInt
import String
import StandardLibrary
import UTF8
import software_amazon_cryptography_primitives_internaldafny_types
import ExternRandom
import Random
import AESEncryption
import ExternDigest
import Digest
import HMAC
import WrappedHMAC
import HKDF
import WrappedHKDF
import Signature
import KdfCtr
import RSAEncryption
import AwsCryptographyPrimitivesOperations
import software_amazon_cryptography_primitives_internaldafny
import Aws_mCryptography
import Aws
import AesKdfCtr

assert "TestAwsCryptographyPrimitivesAesKdfCtr" == __name__
TestAwsCryptographyPrimitivesAesKdfCtr = sys.modules[__name__]

class default__:
    def  __init__(self):
        pass

    @staticmethod
    def AesKdfCtrTests():
        d_0_key_: _dafny.Seq
        d_0_key_ = _dafny.Seq([138, 39, 30, 72, 206, 182, 214, 144, 245, 34, 28, 219, 204, 175, 198, 23, 132, 234, 125, 246, 130, 54, 251, 60, 137, 120, 166, 245, 0, 150, 79, 107])
        d_1_nonce_: _dafny.Seq
        d_1_nonce_ = _dafny.Seq([66, 32, 73, 11, 207, 79, 97, 80, 11, 22, 236, 247, 42, 6, 222, 165])
        d_2_goal_: _dafny.Seq
        d_2_goal_ = _dafny.Seq([143, 128, 174, 191, 9, 171, 140, 22, 40, 143, 11, 239, 249, 101, 61, 120, 176, 23, 233, 210, 125, 72, 114, 70, 209, 170, 206, 96, 24, 112, 215, 169, 100, 136, 162, 231, 208, 24, 135, 121, 223, 13, 239, 180])
        d_3_result_: _dafny.Seq
        d_4_valueOrError0_: Wrappers.Result = Wrappers.Result_Success.default(_dafny.Seq)()
        d_4_valueOrError0_ = AesKdfCtr.default__.AesKdfCtrStream(d_1_nonce_, d_0_key_, 44)
        if not(not((d_4_valueOrError0_).IsFailure())):
            raise _dafny.HaltException("test/TestAesKdfCtr.dfy(18,15): " + _dafny.string_of(d_4_valueOrError0_))
        d_3_result_ = (d_4_valueOrError0_).Extract()
        if not((len(d_3_result_)) == (44)):
            raise _dafny.HaltException("test/TestAesKdfCtr.dfy(19,4): " + _dafny.string_of(_dafny.Seq("expectation violation")))
        if not((d_2_goal_) == (d_3_result_)):
            raise _dafny.HaltException("test/TestAesKdfCtr.dfy(20,4): " + _dafny.string_of(_dafny.Seq("expectation violation")))
        d_0_key_ = _dafny.Seq([212, 191, 10, 32, 13, 55, 22, 101, 189, 182, 186, 119, 202, 16, 175, 49, 103, 82, 87, 190, 13, 142, 103, 201, 98, 84, 228, 47, 142, 96, 61, 167])
        d_1_nonce_ = _dafny.Seq([135, 1, 132, 66, 198, 216, 26, 105, 47, 97, 246, 192, 186, 187, 54, 129])
        d_2_goal_ = _dafny.Seq([11, 154, 37, 42, 116, 143, 238, 68, 62, 135, 225, 71, 98, 224, 135, 18])
        d_5_valueOrError1_: Wrappers.Result = Wrappers.Result_Success.default(_dafny.Seq)()
        d_5_valueOrError1_ = AesKdfCtr.default__.AesKdfCtrStream(d_1_nonce_, d_0_key_, 16)
        if not(not((d_5_valueOrError1_).IsFailure())):
            raise _dafny.HaltException("test/TestAesKdfCtr.dfy(25,11): " + _dafny.string_of(d_5_valueOrError1_))
        d_3_result_ = (d_5_valueOrError1_).Extract()
        if not((len(d_3_result_)) == (16)):
            raise _dafny.HaltException("test/TestAesKdfCtr.dfy(26,4): " + _dafny.string_of(_dafny.Seq("expectation violation")))
        if not((d_2_goal_) == (d_3_result_)):
            raise _dafny.HaltException("test/TestAesKdfCtr.dfy(27,4): " + _dafny.string_of(_dafny.Seq("expectation violation")))
        d_0_key_ = _dafny.Seq([106, 72, 42, 47, 58, 213, 111, 196, 126, 37, 46, 203, 150, 153, 188, 53, 32, 194, 159, 196, 221, 90, 124, 70, 45, 252, 99, 98, 42, 68, 94, 19])
        d_1_nonce_ = _dafny.Seq([13, 247, 32, 159, 220, 254, 69, 218, 42, 110, 159, 42, 209, 244, 79, 232])
        d_2_goal_ = _dafny.Seq([150, 218, 139, 126, 166, 233, 178, 123, 229, 210, 40, 218, 141, 26, 127, 208, 124, 197, 212, 69, 251, 71, 73, 90, 47, 134, 46, 7, 215, 208, 194, 180, 174, 110, 1, 57, 16, 37, 16, 235, 93, 138, 25, 180, 85, 16, 229, 165, 238, 36, 56, 131, 247, 31, 64, 23, 249, 67, 153, 94, 23, 243, 69, 11])
        d_6_valueOrError2_: Wrappers.Result = Wrappers.Result_Success.default(_dafny.Seq)()
        d_6_valueOrError2_ = AesKdfCtr.default__.AesKdfCtrStream(d_1_nonce_, d_0_key_, 64)
        if not(not((d_6_valueOrError2_).IsFailure())):
            raise _dafny.HaltException("test/TestAesKdfCtr.dfy(32,11): " + _dafny.string_of(d_6_valueOrError2_))
        d_3_result_ = (d_6_valueOrError2_).Extract()
        if not((len(d_3_result_)) == (64)):
            raise _dafny.HaltException("test/TestAesKdfCtr.dfy(33,4): " + _dafny.string_of(_dafny.Seq("expectation violation")))
        if not((d_2_goal_) == (d_3_result_)):
            raise _dafny.HaltException("test/TestAesKdfCtr.dfy(34,4): " + _dafny.string_of(_dafny.Seq("expectation violation")))

File: AwsCryptographyPrimitives/runtimes/go/TestsFromDafny-go/TestAwsCryptographyPrimitivesAesKdfCtr/TestAwsCryptographyPrimitivesAesKdfCtr.go
// Package TestAwsCryptographyPrimitivesAesKdfCtr
// Dafny module TestAwsCryptographyPrimitivesAesKdfCtr compiled into Go

package TestAwsCryptographyPrimitivesAesKdfCtr

import (
	os "os"

	m_AESEncryption "github.com/aws/aws-cryptographic-material-providers-library/primitives/AESEncryption"
	m_AesKdfCtr "github.com/aws/aws-cryptographic-material-providers-library/primitives/AesKdfCtr"
	m_AtomicPrimitives "github.com/aws/aws-cryptographic-material-providers-library/primitives/AtomicPrimitives"
	m_AwsCryptographyPrimitivesOperations "github.com/aws/aws-cryptographic-material-providers-library/primitives/AwsCryptographyPrimitivesOperations"
	m_AwsCryptographyPrimitivesTypes "github.com/aws/aws-cryptographic-material-providers-library/primitives/AwsCryptographyPrimitivesTypes"
	m_Digest "github.com/aws/aws-cryptographic-material-providers-library/primitives/Digest"
	m_ECDH "github.com/aws/aws-cryptographic-material-providers-library/primitives/ECDH"
	m_HKDF "github.com/aws/aws-cryptographic-material-providers-library/primitives/HKDF"
	m_HMAC "github.com/aws/aws-cryptographic-material-providers-library/primitives/HMAC"
	m_KdfCtr "github.com/aws/aws-cryptographic-material-providers-library/primitives/KdfCtr"
	m_RSAEncryption "github.com/aws/aws-cryptographic-material-providers-library/primitives/RSAEncryption"
	m_Random "github.com/aws/aws-cryptographic-material-providers-library/primitives/Random"
	m_Signature "github.com/aws/aws-cryptographic-material-providers-library/primitives/Signature"
	m_WrappedHKDF "github.com/aws/aws-cryptographic-material-providers-library/primitives/WrappedHKDF"
	m_WrappedHMAC "github.com/aws/aws-cryptographic-material-providers-library/primitives/WrappedHMAC"
	m_ConstantTime "github.com/aws/aws-cryptographic-material-providers-library/primitives/test/ConstantTime"
	m_ConstantTimeTest "github.com/aws/aws-cryptographic-material-providers-library/primitives/test/ConstantTimeTest"
	m_TestAwsCryptographyPrimitivesDigest "github.com/aws/aws-cryptographic-material-providers-library/primitives/test/TestAwsCryptographyPrimitivesDigest"
	m_TestAwsCryptographyPrimitivesGenerateRandomBytes "github.com/aws/aws-cryptographic-material-providers-library/primitives/test/TestAwsCryptographyPrimitivesGenerateRandomBytes"
	m_TestAwsCryptographyPrimitivesHKDF "github.com/aws/aws-cryptographic-material-providers-library/primitives/test/TestAwsCryptographyPrimitivesHKDF"
	m_TestAwsCryptographyPrimitivesHMacDigest "github.com/aws/aws-cryptographic-material-providers-library/primitives/test/TestAwsCryptographyPrimitivesHMacDigest"
	m_TestECDH "github.com/aws/aws-cryptographic-material-providers-library/primitives/test/TestECDH"
	m_TestKDF "github.com/aws/aws-cryptographic-material-providers-library/primitives/test/TestKDF"
	m_TestKDFK__TestVectors "github.com/aws/aws-cryptographic-material-providers-library/primitives/test/TestKDFK__TestVectors"
	m__System "github.com/dafny-lang/DafnyRuntimeGo/v4/System_"
	_dafny "github.com/dafny-lang/DafnyRuntimeGo/v4/dafny"
	m_Actions "github.com/dafny-lang/DafnyStandardLibGo/Actions"
	m_Base64 "github.com/dafny-lang/DafnyStandardLibGo/Base64"
	m_Base64Lemmas "github.com/dafny-lang/DafnyStandardLibGo/Base64Lemmas"
	m_BoundedInts "github.com/dafny-lang/DafnyStandardLibGo/BoundedInts"
	m_DivInternals "github.com/dafny-lang/DafnyStandardLibGo/DivInternals"
	m_DivInternalsNonlinear "github.com/dafny-lang/DafnyStandardLibGo/DivInternalsNonlinear"
	m_DivMod "github.com/dafny-lang/DafnyStandardLibGo/DivMod"
	m_FileIO "github.com/dafny-lang/DafnyStandardLibGo/FileIO"
	m_FloatCompare "github.com/dafny-lang/DafnyStandardLibGo/FloatCompare"
	m_Functions "github.com/dafny-lang/DafnyStandardLibGo/Functions"
	m_GeneralInternals "github.com/dafny-lang/DafnyStandardLibGo/GeneralInternals"
	m_GetOpt "github.com/dafny-lang/DafnyStandardLibGo/GetOpt"
	m_HexStrings "github.com/dafny-lang/DafnyStandardLibGo/HexStrings"
	m_Logarithm "github.com/dafny-lang/DafnyStandardLibGo/Logarithm"
	m__Math "github.com/dafny-lang/DafnyStandardLibGo/Math_"
	m_ModInternals "github.com/dafny-lang/DafnyStandardLibGo/ModInternals"
	m_ModInternalsNonlinear "github.com/dafny-lang/DafnyStandardLibGo/ModInternalsNonlinear"
	m_Mul "github.com/dafny-lang/DafnyStandardLibGo/Mul"
	m_MulInternals "github.com/dafny-lang/DafnyStandardLibGo/MulInternals"
	m_MulInternalsNonlinear "github.com/dafny-lang/DafnyStandardLibGo/MulInternalsNonlinear"
	m_Power "github.com/dafny-lang/DafnyStandardLibGo/Power"
	m_Relations "github.com/dafny-lang/DafnyStandardLibGo/Relations"
	m_Seq "github.com/dafny-lang/DafnyStandardLibGo/Seq"
	m_Seq_MergeSort "github.com/dafny-lang/DafnyStandardLibGo/Seq_MergeSort"
	m_Sorting "github.com/dafny-lang/DafnyStandardLibGo/Sorting"
	m_StandardLibrary "github.com/dafny-lang/DafnyStandardLibGo/StandardLibrary"
	m_StandardLibraryInterop "github.com/dafny-lang/DafnyStandardLibGo/StandardLibraryInterop"
	m_StandardLibrary_String "github.com/dafny-lang/DafnyStandardLibGo/StandardLibrary_String"
	m_StandardLibrary_UInt "github.com/dafny-lang/DafnyStandardLibGo/StandardLibrary_UInt"
	m_Streams "github.com/dafny-lang/DafnyStandardLibGo/Streams"
	m_UnicodeStrings "github.com/dafny-lang/DafnyStandardLibGo/UnicodeStrings"
	m__Unicode "github.com/dafny-lang/DafnyStandardLibGo/Unicode_"
	m_Utf16EncodingForm "github.com/dafny-lang/DafnyStandardLibGo/Utf16EncodingForm"
	m_Utf8EncodingForm "github.com/dafny-lang/DafnyStandardLibGo/Utf8EncodingForm"
	m_Wrappers "github.com/dafny-lang/DafnyStandardLibGo/Wrappers"
)

var _ = os.Args
var _ _dafny.Dummy__
var _ m__System.Dummy__
var _ m_Wrappers.Dummy__
var _ m_BoundedInts.Dummy__
var _ m_StandardLibrary_UInt.Dummy__
var _ m_StandardLibrary_String.Dummy__
var _ m_StandardLibrary.Dummy__
var _ m_AwsCryptographyPrimitivesTypes.Dummy__
var _ m_Random.Dummy__
var _ m_AESEncryption.Dummy__
var _ m_Digest.Dummy__
var _ m_HMAC.Dummy__
var _ m_WrappedHMAC.Dummy__
var _ m_HKDF.Dummy__
var _ m_WrappedHKDF.Dummy__
var _ m_Signature.Dummy__
var _ m_KdfCtr.Dummy__
var _ m_RSAEncryption.Dummy__
var _ m_ECDH.Dummy__
var _ m_AwsCryptographyPrimitivesOperations.Dummy__
var _ m_AtomicPrimitives.Dummy__
var _ m_Relations.Dummy__
var _ m_Seq_MergeSort.Dummy__
var _ m__Math.Dummy__
var _ m_Seq.Dummy__
var _ m__Unicode.Dummy__
var _ m_Functions.Dummy__
var _ m_Utf8EncodingForm.Dummy__
var _ m_Utf16EncodingForm.Dummy__
var _ m_UnicodeStrings.Dummy__
var _ m_FileIO.Dummy__
var _ m_GeneralInternals.Dummy__
var _ m_MulInternalsNonlinear.Dummy__
var _ m_MulInternals.Dummy__
var _ m_Mul.Dummy__
var _ m_ModInternalsNonlinear.Dummy__
var _ m_DivInternalsNonlinear.Dummy__
var _ m_ModInternals.Dummy__
var _ m_DivInternals.Dummy__
var _ m_DivMod.Dummy__
var _ m_Power.Dummy__
var _ m_Logarithm.Dummy__
var _ m_StandardLibraryInterop.Dummy__
var _ m_Streams.Dummy__
var _ m_Sorting.Dummy__
var _ m_HexStrings.Dummy__
var _ m_GetOpt.Dummy__
var _ m_FloatCompare.Dummy__
var _ m_Base64.Dummy__
var _ m_Base64Lemmas.Dummy__
var _ m_Actions.Dummy__
var _ m_TestKDF.Dummy__
var _ m_TestKDFK__TestVectors.Dummy__
var _ m_TestAwsCryptographyPrimitivesHMacDigest.Dummy__
var _ m_TestECDH.Dummy__
var _ m_TestAwsCryptographyPrimitivesHKDF.Dummy__
var _ m_TestAwsCryptographyPrimitivesGenerateRandomBytes.Dummy__
var _ m_TestAwsCryptographyPrimitivesDigest.Dummy__
var _ m_ConstantTime.Dummy__
var _ m_ConstantTimeTest.Dummy__

type Dummy__ struct{}

// Definition of class Default__
type Default__ struct {
	dummy byte
}

func New_Default___() *Default__ {
	_this := Default__{}

	return &_this
}

type CompanionStruct_Default___ struct {
}

var Companion_Default___ = CompanionStruct_Default___{}

func (_this *Default__) Equals(other *Default__) bool {
	return _this == other
}

func (_this *Default__) EqualsGeneric(x interface{}) bool {
	other, ok := x.(*Default__)
	return ok && _this.Equals(other)
}

func (*Default__) String() string {
	return "TestAwsCryptographyPrimitivesAesKdfCtr.Default__"
}
func (_this *Default__) ParentTraits_() []*_dafny.TraitID {
	return [](*_dafny.TraitID){}
}

var _ _dafny.TraitOffspring = &Default__{}

func (_static *CompanionStruct_Default___) AesKdfCtrTests() {
	var _0_key _dafny.Sequence
	_ = _0_key
	_0_key = _dafny.SeqOf(uint8(138), uint8(39), uint8(30), uint8(72), uint8(206), uint8(182), uint8(214), uint8(144), uint8(245), uint8(34), uint8(28), uint8(219), uint8(204), uint8(175), uint8(198), uint8(23), uint8(132), uint8(234), uint8(125), uint8(246), uint8(130), uint8(54), uint8(251), uint8(60), uint8(137), uint8(120), uint8(166), uint8(245), uint8(0), uint8(150), uint8(79), uint8(107))
	var _1_nonce _dafny.Sequence
	_ = _1_nonce
	_1_nonce = _dafny.SeqOf(uint8(66), uint8(32), uint8(73), uint8(11), uint8(207), uint8(79), uint8(97), uint8(80), uint8(11), uint8(22), uint8(236), uint8(247), uint8(42), uint8(6), uint8(222), uint8(165))
	var _2_goal _dafny.Sequence
	_ = _2_goal
	_2_goal = _dafny.SeqOf(uint8(143), uint8(128), uint8(174), uint8(191), uint8(9), uint8(171), uint8(140), uint8(22), uint8(40), uint8(143), uint8(11), uint8(239), uint8(249), uint8(101), uint8(61), uint8(120), uint8(176), uint8(23), uint8(233), uint8(210), uint8(125), uint8(72), uint8(114), uint8(70), uint8(209), uint8(170), uint8(206), uint8(96), uint8(24), uint8(112), uint8(215), uint8(169), uint8(100), uint8(136), uint8(162), uint8(231), uint8(208), uint8(24), uint8(135), uint8(121), uint8(223), uint8(13), uint8(239), uint8(180))
	var _3_valueOrError0 m_Wrappers.Result = m_Wrappers.Companion_Result_.Default(_dafny.EmptySeq)
	_ = _3_valueOrError0
	_3_valueOrError0 = m_AesKdfCtr.AesKdfCtrStream(_1_nonce, _0_key, uint32(44))
	if !(!((_3_valueOrError0).IsFailure())) {
		panic("test/TestAesKdfCtr.dfy(18,18): " + (_3_valueOrError0).String())
	}
	var _4_result _dafny.Sequence
	_ = _4_result
	_4_result = (_3_valueOrError0).Extract().(_dafny.Sequence)
	if !((_dafny.IntOfUint32((_4_result).Cardinality())).Cmp(_dafny.IntOfInt64(44)) == 0) {
		panic("test/TestAesKdfCtr.dfy(19,4): " + (_dafny.SeqOfString("expectation violation")).String())
	}
	if !(_dafny.Companion_Sequence_.Equal(_2_goal, _4_result)) {
		panic("test/TestAesKdfCtr.dfy(20,4): " + (_dafny.SeqOfString("expectation violation")).String())
	}
	_0_key = _dafny.SeqOf(uint8(212), uint8(191), uint8(10), uint8(32), uint8(13), uint8(55), uint8(22), uint8(101), uint8(189), uint8(182), uint8(186), uint8(119), uint8(202), uint8(16), uint8(175), uint8(49), uint8(103), uint8(82), uint8(87), uint8(190), uint8(13), uint8(142), uint8(103), uint8(201), uint8(98), uint8(84), uint8(228), uint8(47), uint8(142), uint8(96), uint8(61), uint8(167))
	_1_nonce = _dafny.SeqOf(uint8(135), uint8(1), uint8(132), uint8(66), uint8(198), uint8(216), uint8(26), uint8(105), uint8(47), uint8(97), uint8(246), uint8(192), uint8(186), uint8(187), uint8(54), uint8(129))
	_2_goal = _dafny.SeqOf(uint8(11), uint8(154), uint8(37), uint8(42), uint8(116), uint8(143), uint8(238), uint8(68), uint8(62), uint8(135), uint8(225), uint8(71), uint8(98), uint8(224), uint8(135), uint8(18))
	var _5_valueOrError1 m_Wrappers.Result = m_Wrappers.Companion_Result_.Default(_dafny.EmptySeq)
	_ = _5_valueOrError1
	_5_valueOrError1 = m_AesKdfCtr.AesKdfCtrStream(_1_nonce, _0_key, uint32(16))
	if !(!((_5_valueOrError1).IsFailure())) {
		panic("test/TestAesKdfCtr.dfy(25,14): " + (_5_valueOrError1).String())
	}
	_4_result = (_5_valueOrError1).Extract().(_dafny.Sequence)
	if !((_dafny.IntOfUint32((_4_result).Cardinality())).Cmp(_dafny.IntOfInt64(16)) == 0) {
		panic("test/TestAesKdfCtr.dfy(26,4): " + (_dafny.SeqOfString("expectation violation")).String())
	}
	if !(_dafny.Companion_Sequence_.Equal(_2_goal, _4_result)) {
		panic("test/TestAesKdfCtr.dfy(27,4): " + (_dafny.SeqOfString("expectation violation")).String())
	}
	_0_key = _dafny.SeqOf(uint8(106), uint8(72), uint8(42), uint8(47), uint8(58), uint8(213), uint8(111), uint8(196), uint8(126), uint8(37), uint8(46), uint8(203), uint8(150), uint8(153), uint8(188), uint8(53), uint8(32), uint8(194), uint8(159), uint8(196), uint8(221), uint8(90), uint8(124), uint8(70), uint8(45), uint8(252), uint8(99), uint8(98), uint8(42), uint8(68), uint8(94), uint8(19))
	_1_nonce = _dafny.SeqOf(uint8(13), uint8(247), uint8(32), uint8(159), uint8(220), uint8(254), uint8(69), uint8(218), uint8(42), uint8(110), uint8(159), uint8(42), uint8(209), uint8(244), uint8(79), uint8(232))
	_2_goal = _dafny.SeqOf(uint8(150), uint8(218), uint8(139), uint8(126), uint8(166), uint8(233), uint8(178), uint8(123), uint8(229), uint8(210), uint8(40), uint8(218), uint8(141), uint8(26), uint8(127), uint8(208), uint8(124), uint8(197), uint8(212), uint8(69), uint8(251), uint8(71), uint8(73), uint8(90), uint8(47), uint8(134), uint8(46), uint8(7), uint8(215), uint8(208), uint8(194), uint8(180), uint8(174), uint8(110), uint8(1), uint8(57), uint8(16), uint8(37), uint8(16), uint8(235), uint8(93), uint8(138), uint8(25), uint8(180), uint8(85), uint8(16), uint8(229), uint8(165), uint8(238), uint8(36), uint8(56), uint8(131), uint8(247), uint8(31), uint8(64), uint8(23), uint8(249), uint8(67), uint8(153), uint8(94), uint8(23), uint8(243), uint8(69), uint8(11))
	var _6_valueOrError2 m_Wrappers.Result = m_Wrappers.Companion_Result_.Default(_dafny.EmptySeq)
	_ = _6_valueOrError2
	_6_valueOrError2 = m_AesKdfCtr.AesKdfCtrStream(_1_nonce, _0_key, uint32(64))
	if !(!((_6_valueOrError2).IsFailure())) {
		panic("test/TestAesKdfCtr.dfy(32,14): " + (_6_valueOrError2).String())
	}
	_4_result = (_6_valueOrError2).Extract().(_dafny.Sequence)
	if !((_dafny.IntOfUint32((_4_result).Cardinality())).Cmp(_dafny.IntOfInt64(64)) == 0) {
		panic("test/TestAesKdfCtr.dfy(33,4): " + (_dafny.SeqOfString("expectation violation")).String())
	}
	if !(_dafny.Companion_Sequence_.Equal(_2_goal, _4_result)) {
		panic("test/TestAesKdfCtr.dfy(34,4): " + (_dafny.SeqOfString("expectation violation")).String())
	}
}

// End of class Default__
